Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -814,6 +816,14 @@ public async Task<SandboxedProcessPipExecutionResult> RunAsync(CancellationToken
             }
         }
 
+        private SharedOpaqueJournal CreateSharedOpaqueJournalIfConfigured()
+        {
+            // don't use this journal if pip's semistable hash is 0 or if journal root directory is not set up

```

Comment:
>if pip's semistable hash is 0 [](start = 38, length = 29)

What is the meaning of a process pip with semistable hash 0?

Reply:
Some tests use that and I wanted to avoid running this on those tests. 

What ends up happening, in a single test there might be multiple pips with semi stable hash 0, and then we end up creating multiple journal objects concurrently writing to the same journal file (which is not good)

Reply:
added a comment about it